### ISSUE TITLE
Rocket entry! :)

### ISSUE THREAD

**Issue by Tyrantin** [zmaster587/AdvancedRocketry#434]
A Bridge function that allows you to activate  putting out a ramp to the Rocket. Some servers do not allow fly mode. this would assist player in the mod also allowing cool functions.

**Comment by zmaster587** [comment]
While that shouldn't be an issue because you can get on the rocket as long
as you can touch any part of it, a bridge would be epic. Could also use
bridges to function as airlocks etc

**Comment by Tyrantin** [comment]
Exactly! Clicking on a rocket that is 25 blocks high is kind of unorthodox! Do Astronauts Have to climb a rocket before they go into space! Nahhh... Give some reality. I'm not saying that the "Ramp" Has to be built to the exact level of the "Seat". That could be animated with the tower. It would require a "Fue" lines of code to scan the seat location. :) No I don't know code but I understand the concept.

**Comment by zmaster587** [comment]
Oh, I thought you were saying I should put something like drawbridges from TiC to get onto the rocket.  Adding a mount/dismount point and a way to climb it would be much harder...
What I'd like to eventually do is have decent collision for the rocket so you can go inside it without dissassembly, once that is implemented it would be trival to make player built interiors/exteriors with ramps/etc.

**Comment by Tyrantin** [comment]
Basically you could use the Towers to "Examine" the rocket to find the Hatch. The system would then have to align to the hatch for the ramp to extend to the ship! The examination could be part of the "Build" function of the rocket. The hatch would have to be part of the Player launch rocket. In basic the platform would be "Sort of" like the TiC only redesigned to fit the Theme of AR. Just an FYI I am currently building a Mod Pack set around AR and Matter Overdrive! :)

**Comment by Joey-Destroyah** [comment]
> Oh, I thought you were saying I should put something like drawbridges from TiC to get onto the rocket. Adding a mount/dismount point and a way to climb it would be much harder... What I'd like to eventually do is have decent collision for the rocket so you can go inside it without dissassembly, once that is implemented it would be trival to make player built interiors/exteriors with ramps/etc.
> 
> I'm not quite sure how I could make a general purpose exterior ramp. Because rockets could some in so many shapes and sizes, and just making the whole think climbable would be rather insane. I'll see what I can do though. Part of implementing interiors would allow for pistons and ladder to be made part of the rocket.

Maybe on a launch pad like LC39A, there could be an elevator from Dynamic transportation.

**Comment by voidsong-dragonfly** [comment]
Rocket collision is pain, this got extremely complicated with later versions. Once again like your last issue comment, we are NOT integrating with random mods unless there is very good reason to

**Comment by Joey-Destroyah** [comment]
Ok. I used dynamic transport as an example, but I get your point.
